brainglobe_atlasapi/atlas_generation/atlas_scripts/perens_stereotaxic_mri_mouse.py

```diff
@@ -0,0 +1,325 @@
+__version__ = "0"
+import json
+import multiprocessing as mp
+import os
+import time
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import py7zr
+from brainglobe_utils.IO.image import load_any
+from rich.progress import track
+
+from brainglobe_atlasapi import BrainGlobeAtlas, utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+atlas = BrainGlobeAtlas("allen_mouse_25um")
```

Comment:
I would maybe rename this and add a comment for legibility? So we avoid confusion between the atlas packaged here and the Allen one? 
```suggestion
# The Perens atlas re-uses information from the Allen atlas, so it's useful to have an instance of the Allen atlas around
allen_atlas = BrainGlobeAtlas("allen_mouse_25um")
```